conftest.py

```diff
@@ -247,9 +259,18 @@ def pytest_cmdline_main(config):
             if config.getoption("cnv_source") != "production":
                 raise ValueError("Missing --cnv-image")
 
+    if upgrade_option == "eus":
+        eus_ocp_images = config.getoption("eus_ocp_images")
+        if not eus_ocp_images:
+            raise ValueError("Missing --eus-ocp-images")
+        if len(eus_ocp_images.split(",")) != 2:
+            raise ValueError(
+                f"Two OCP images are needed to perform EUS-to-EUS upgrade.Provided images: {eus_ocp_images}"
+            )
```

Comment:
You can combine both into one check and please add a space after "." for the second error.